src/include/ndpi_protocol_ids.h

```diff
@@ -334,6 +334,20 @@ typedef enum {
   NDPI_PROTOCOL_PSIPHON               = 303,
   NDPI_PROTOCOL_ULTRASURF             = 304,
   NDPI_PROTOCOL_THREEMA               = 305,
+  NDPI_PROTOCOL_CYBERREEF             = 306,
```

Comment:
Thanks but that was intended to be a throw-away proof-of-concept that we knew how to test.  Please remove it.